part2/phonebook/src/components/PersonList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import Person from './Person.js'
+
+const PersonList = ({personList}) => {
+  const personToShow = personList
+
+  return (
+    <div>
+      {personToShow.map(person => 
+        <Person key={person.id} person={person} />
+      )}
+    </div>
```

Comment:
why not just personList.map ?

Reply:
forgot that the prop was the variable i was using, dont need to rename it to personToShow.